src/build.jl

```diff
@@ -101,40 +101,56 @@ function tau(S::ScalarSector,H::ScalarConsumer)
     -sum( compensated_demand(netput) * total_tax(netput, H) * commodity(netput) for (_,N)∈netputs(P) for netput∈N if total_tax(netput,H)!=0; init=0)
 end
 
```

Comment:
This (tau) is redundant now, right? Delete, and update the export and the reference in the documentation?